FND/Organizations/Organizations.rdf

```diff
@@ -103,5 +101,22 @@
 		<fibo-fnd-utl-av:adaptedFrom>https://www.iso.org/obp/ui/#iso:std:iso-iec:6523:-1:ed-1:v1:en</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:adaptedFrom>https://www.w3.org/TR/vocab-org/#class-organization</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-org-org;OrganizationMember">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&lcc-lr;isMemberOf"/>
+						<owl:allValuesFrom rdf:resource="&fibo-fnd-org-org;Organization"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>organization member</rdfs:label>
+		<skos:definition>identifies an organization or someone that plays a membership role in some organization</skos:definition>
```

Comment:
No this IS the membership role. The property isPlayedBy identifies who plays the role.

Reply:
> No this IS the membership role. The property isPlayedBy identifies who plays the role.

Yes, you're right - I didn't want to change everything about the class in this issue, just move it to untangle the circular and reversed dependencies between parties, people, and organizations.